python/lsst/ts/standardscripts/auxtel/integration_tests/dometrajectory_mcs.py

```diff
@@ -487,9 +487,12 @@ async def _track_telescope_loop(self):
                 # TODO DM-24051: ditch this hack
                 # when we no longer need ts_xml 4.8
                 if hasattr(self.atmcs.cmd_trackTarget.DataType(), "taiTime"):
+                    self.log.info("taiTime")
```

Comment:
Thanks for the TODO. Have we reached this point yet or are we still using XML 4.8?